composer.json

```diff
@@ -10,12 +10,15 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
-        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0",
-        "symfony/filesystem": "^4.4 || ^5.3 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.3 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.3 || ^6.0",
-        "symfony/yaml": "^4.4 || ^5.3 || ^6.0"
+        "php": "^7.2 || ^8.0 || ^8.1",
+        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0 || ^6.1",
```

Comment:
Not sure if we need to add also 5.4 to the list?

Reply:
Na, no need